databroker/mongo_normalized.py

```diff
@@ -261,6 +272,20 @@ def metadata(self):
             transformed["stop"] = self.transforms["stop"](self._metadata["stop"])
         metadata = dict(collections.ChainMap(transformed, self._metadata))
         return metadata
+    
+    async def update_metadata(self, metadata=None, specs=None):
+        # if(specs is not None or "start" not in metadata):
```

Comment:
As written this silently ignore any input to `specs`. The "ignores" part of this is fine for the foreseeable future, but the "silently" part could conceivably give someone a bad day.

Can we get `if specs is None` and some kind of "Updating specs is not yet supported" error here?

Reply:
resolved in 01a556cc89b9b019bbd5bec2e044d5af39605665
So long as you meant `if specs is not None` which would mean, "If specs is present, let the user know they cannot update it"